src/System.Net.Http/src/System/Net/Http/Unix/HttpClientHandler.Unix.cs

```diff
@@ -0,0 +1,147 @@
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public DecompressionMethods AutomaticDecompression
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public bool UseProxy
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public IWebProxy Proxy
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public bool PreAuthenticate
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public bool UseDefaultCredentials
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public ICredentials Credentials
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public bool AllowAutoRedirect
+        {
+            get
+            {
+                return _curlHandler.AutomaticRedirection;
+            }
+            set
+            {
+                _curlHandler.AutomaticRedirection = value;
+            }
+        }
+
+        public int MaxAutomaticRedirections
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public long MaxRequestContentBufferSize
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        #endregion Properties
+
+        #region De/Constructors
+
+        public HttpClientHandler()
+        {
+            _curlHandler = new CurlHandler();
+
+            // TODO: Set same defaults as Windows handler
+            AllowAutoRedirect = true;
+        }
+
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing && !_disposed)
+            {
+                _disposed = true;
```

Comment:
```_curlHandler``` is itself already tracking whether it's been disposed.  Does this wrapper then also need to do such tracking, or could we remove the ```_disposed``` field from this type entirely?